.github/workflows/release-tag.yaml

```
@@ -15,14 +15,8 @@ jobs:
         with:
           fetch-depth: 0
-      - name: Dotenv Action
-        id: load-env
-        uses: falti/dotenv-action@v1.1.4
-        with:
-          path: .env.ci
-      - uses: subosito/flutter-action@v2
-        with:
-          flutter-version: ${{ steps.load-env.outputs.flutter_version }}
+      - name: Flutter setup
+        uses: ./.github/actions/flutter-setup
```

actionlint (1.7.4)

description is required in metadata of "Setup Flutter Environment from environment" action at "/inmem/16/10cd492c-1bc2-43e1-b5b6-ed8bcff62cc0/home/jailuser/git/.github/actions/flutter-setup/action.yaml"

(action)